src/Notes/NavigationNudge.php

```diff
@@ -30,6 +30,9 @@ class NavigationNudge {
 	 * Attach hooks.
 	 */
 	public function __construct() {
+		if ( ! Features::exists( 'navigation' ) ) {
+			self::action_note( '', 'yes' );
```

Comment:
I think instead of actioning the note (since it actually was not actioned, maybe we could delete the note from the data store.

`Notes::delete_notes_with_name( self::NOTE_NAME );`

Reply:
If we delete the note, will the note be recreated if Navigation is enabled again in the future, in `possible_add_note()`?

As long as that will work, I agree that it seems more correct to actually delete the note rather than actioning it.

Reply:
I had a version of this that used `self::possibly_delete_note()` but that removed it from the database entirely rather than setting the `is_deleted` flag like I expected so I went back to actioning. I'll look into `Notes::delete_notes_with_name()`.

Reply:
Setting the `is_deleted` flag is a better idea if we want the note to not show up again, but I would think this note should (re)appear if the feature is available in the currently active version.